bar.go

```diff
@@ -54,8 +55,8 @@ func barCreateIcon(taskbar bool, idata *IconData, win Window) *barIcon {
 }
 
 func (bsl *barStackListener) WindowAdded(win Window) {
-	fdoDesktop := FdoLookupApplicationWinInfo(win.Title(), win.Class(), win.Command(), win.IconName())
-	icon := barCreateIcon(true, fdoDesktop, win)
+	idata := driver.GetIconDataByWinInfo(iconTheme, iconSize, win.Title(), win.Class(), win.Command(), win.IconName())
```

Comment:
I'm not sure about the "i" prefix here... is it for interface? if so we don't do that in Go.
If it's part of the name then maybe "data" or "iconData" is better.